README.rst

```diff
@@ -23,15 +23,16 @@ Additionally this program acts as a basic example of using the `pysunspec`_ Pyth
 Installation
 ------------
 
-While ``boots.py`` can be run with Python 2.7 and 3.4+, it uses Python 3.7 to create a venv.
-
+Must install poetry & poetry-dynamic-versionioning
+pip install poetry
+pip install poetry-dynamic-versioning
 
 Windows
 =======
 
 ::
 
-    py boots.py ensure
+    poetry install
     venv\Scripts\epcsunspecdemo get-models
```

Comment:
Is the virtual environment directory named `venv`? My limited knowledge suggests it is `.venv` or possibly configurable?

Reply:
Looks like it was replaced with `poetry run`.